src/Utils/samples.jl

```diff
@@ -344,6 +344,85 @@ function sample!(D::Vector{VN}, X::LazySet, sampler::CombinedSampler;
     throw(ErrorException("sampling failed"))
 end
 
+"""
+    FaceSampler <: AbstractSampler
+
+Type used for sampling from the `k`-faces of a set.
+
+### Fields
+
+- `dim` -- dimension of the faces to be sampled; a negative number is
+           interpreted as `n - dim` where `n` is the dimension of the set
+
+### Notes
+
+For a three-dimensional polytope, the following face dimensions exist:
+- 3-face – the polytope itself
+- 2-faces – 2-dimensional polygonal faces
+- 1-faces – 1-dimensional edges
+- 0-faces – 0-dimensional vertices
+
+For more information see
+[Wikipedia](https://en.wikipedia.org/wiki/Face_(geometry)#k-face).
+"""
```

Comment:
```suggestion
````julia
### Algorithm

For each point to be sampled, we randomly split the integers `1 .. n` into two subgroups of size `k` and `n-k` respectively. For the i-th coordinate in the first group, we sample in the interval `low(H, i) .. high(H, i)`. For the i-th coordinate in the second group, we randomly pick either `low(H, i)` or `high(H, i)`. 
"""
````